src/libs/actions/Session/index.js

```diff
@@ -446,13 +470,13 @@ export {
     updatePasswordAndSignin,
     signIn,
     signInWithShortLivedAuthToken,
+    cleanupSession,
```

Comment:
NAB but this does not really need to be moved.

Reply:
Just a bad revert job from me 😅